hedera-node/hedera-app/src/main/java/com/hedera/node/app/grpc/impl/netty/GrpcServiceBuilder.java

```diff
@@ -115,14 +116,14 @@ final class GrpcServiceBuilder {
      */
     public GrpcServiceBuilder(
             @NonNull final String serviceName,
-            @NonNull final IngestWorkflow ingestWorkflow,
+            @Nullable final IngestWorkflow ingestWorkflow,
```

Comment:
Imo it makes more sense to keep this `@NonNull` since this builder **must** have a non-null `IngestWorkflow` for HAPI and it's harmless to construct the query-only node operator `Server` with a reference to the `IngestWorkflow`

Reply:
I wanted to make sure that the query-only node operator Server will not be able to execute transactions, but yes now it seems like the HAPI one might accept null `IngestWorkflow`. I will revert it back to `@NonNull`.